tests/test_text.py

```diff
@@ -1,20 +1,41 @@
-import pytest
 import numpy as np
-
-from spacy.vocab import Vocab
+import pytest
+from gensim.models import Word2Vec
+from gensim.utils import tokenize
 from spacy.language import Language
+from spacy.vocab import Vocab
 
-from embetter.text import SentenceEncoder, BytePairEncoder, spaCyEncoder
+from embetter.text import (
+    BytePairEncoder,
+    SentenceEncoder,
+    Word2VecEncoder,
+    spaCyEncoder,
+)
 from embetter.utils import cached
 
-
 test_sentences = [
     "This is a test sentence!",
     "And this is another one",
     "\rUnicode stuff: ♣️,♦️,❤️,♠️\n",
 ]
 
 
+@pytest.mark.parametrize("setting", ["max", "mean", "both"])
+def test_word2vec(setting):
+    """Check if one can train and use a very simple word embedding model."""
+    vector_size = 25
+    sentences = [list(tokenize(sent)) for sent in test_sentences]
+    model = Word2Vec(
+        sentences=sentences, vector_size=vector_size, window=3, min_count=1
+    )
+    encoder = Word2VecEncoder(model, agg=setting)
```

Comment:
Might also be good to save the `model` in a tempdir and to confirm that the `Word2VecEncoder` can actually read it from disk as well.